v3/docs/00-style-and-contributor-guidelines/a-contributor-guidelines/index.mdx

```diff
@@ -0,0 +1,189 @@
+
+### How-to guide template
+
+We recommend you use the how-to guide template to structure articles that you want to submit as to “how-to” topics.
+You can download a copy of the Markdown template directly from [here](how-to-template.md).
+
+After downloading the template, rename the file and replace the description of each section with the relevant content.
+
+----
+
+#### [Guide title]
+
+The guide title should summarize the goal of the article. For “how-to” guides, the title should complete the “How do I …?” sentence.
+
+#### Overview
+
+The first paragraph of the article should provide a brief overview of what the article is about and why this information is useful to its audience. 
+The overview section does not require an **Overview** heading and it might be more than one paragraph.
+
+The opening section of each article sets the stage for what follows and should answer the obvious questions so readers can decide whether the content is relevant to them. 
+Readers should know—just from reading this section—whether they should continue or the content doesn’t apply to them and they should move on to something else.
+
+For your overview, try to answer the following questions:
+
+* What is this article about?
+
+* What is the purpose or goal to be accomplished by following the procedure or technique?
+
+* Why would someone want to use this procedure or technique? For example, are there specific use case scenarios that are applicable?
+
+* When would someone use this procedure? For example, is this an activity that is done once or repeated? Is it a pattern or a unique case?
+
+* Where is the procedure or technique applicable?
+
+* Who would use this procedure or technique? For example, are special skills required? Do specific permissions or restrictions apply?
+
+The overview section is also a good place to link to other resources, including other guides. 
+As the content creator, you want readers to have confidence that the guide will be useful for them.
+
+#### Use cases
+
+This section is optional because the guide title might be sufficient if your article is tightly focused on a single use case.
+
+If your article has more than one practical application, use this section to briefly describe each of them.
+
+If the only use case is a repetition of the article title or sufficiently covered in the overview section, skip this section.
+
+If your article only has one use case but it requires more explanation than the title provides, add this section and one or more sentences to provide the additional explanation. For example:
+
+* This guide illustrates implementing a second currency for paying fees. If you want to support multiple currencies in your runtime, this guide provides practical advice and detailed steps that you can apply for the additional currencies you want to support.
+
+* This guide shows you how to perform a runtime migration from a Vec<u32> to SomeStruct
```

Comment:
@sacha-l 
MDX is throwing an error here because it thinks it's an HTML tag without a closing tag.  Either wrap it in code or perhaps a string.